CHANGELOG.md

```diff
@@ -1,18 +1,36 @@
 Here we document changes that affect the public API or changes that needs to be communicated to other developers. 
 
+## 2019-09-02 New Inviwo Version 0.9.11
```

Comment:
Is this sentence grammatically sound? Maybe change 'New' to 'Release of'

Reply:
in line with prior releases ;-)
e.g. `## 2019-07-04 New Inviwo version 0.9.10`

Reply:
Well, wouldn't have hurt if we changed the prior releases as well